src/tests/c_guests/c_callbackguest/main.c

```diff
@@ -1,5 +1,6 @@
 // Included from hyperlight_guest_capi/include
 #include "hyperlight_guest.h"
+// TODO(danbugs): update when I'm done w/ the move
```

Comment:
Does this comment need action or deletion?

Reply:
I addressed all TODOs by the end of the PR.